Dockerfile.local

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 PingCAP, Inc.
```

Comment:
```suggestion
# Copyright 2021 PingCAP, Inc.
```

Reply:
why adding this file to this PR?

Reply:
Sorry, this was accidental commit, deleted.